modules/op-running-unprivileged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,136 @@
+
+.Prerequisites
+
+. Ensure that the container is configured properly by associating the `/etc/subuid` and `/etc/subgid` files with the latest `shadow-utils` package.
+
+. Create a `ConfigMap` link:https://kubernetes.io/docs/concepts/storage/volumes/#configmap[storage volume] that the build can use as a workspace.
+
+.Procedure
+
+. Use the publicly available link:https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile] image, which is based on the Red Hat Universal Base Image (UBI).
+
+. To prevent the `buildah` container from being run randomly, force run it as the `build` user, without the setup necessary to run as a non-root user. 
++
+[NOTE]
+====
+If you set up a different user, you need to force run the container as this user with necessary `subuid` and `subgid` configuration.
+====
+
+. Add the security context to each build and push steps.
+
+.. Export the `buildah` cluster task as a task to the project's namepsace.
++
+[source,terminal]
+----
+$ tkn task create --from=buildah
+----
+
+.. Edit the `buildah` task within the project's namespace:
++
+[source,terminal]
+----
+$ oc edit task buildah
+----
+
+.. Add the security context.
++
+[source,yaml,subs="attributes+"]
+----
+...
+securityContext:
+    runAsUser: 1000
+...
+----
+
+. Change the `mountPath` so that the build uses the local container directory of the new user:
++
+[source,yaml,subs="attributes+"]
+----
+...
+volumeMounts:
+    - name: varlibcontainers
+      mountPath: /home/build/.local/share/containers
+...
+----
+
+. Verify that the image runs as a user.
++
+[source,terminal,subs="attributes+"]
+----
+$ tkn tr logs -Lf buildah-run
+----
++
+.Example output
+[source,terminal]
+----
+[build] ++ id
+[build] Running as USER ID uid=1000(build) gid=1000(build) groups=1000(build),1000690000
+[build] + echo 'Running as USER ID uid=1000(build) gid=1000(build) groups=1000(build),1000690000'
+[build] + buildah --storage-driver=vfs bud --format=oci --tls-verify=true --no-cache -f ./Dockerfile -t image-registry.openshift-image-registry.svc:5000/test/buildahuser .
+[build] STEP 1: FROM registry.access.redhat.com/ubi8/ubi AS buildah-runner
+[build] Getting image source signatures
+[build] Checking if image destination supports signatures
+[build] Copying blob sha256:adffa69631469a649556cee5b8456f184928818064aac82106bd08bd62e51d4e
+[build] Copying blob sha256:26f1167feaf74177f9054bf26ac8775a4b188f25914e23bda9574ef2a759cce4
+[build] Copying config sha256:fca12da1dc30ed8e7d03afb84b287fc695673fff9c04bfcb2ff404b558670a36
+[build] Writing manifest to image destination
+[build] Storing signatures
+[build] STEP 2: RUN dnf -y update &&     dnf -y install git &&     dnf clean all
+[build] Updating Subscription Management repositories.
+[...]
+----
+
+For reference, you can use the following `ConfigMap` object with sample `Dockerfile` and `TaskRun` objects:
```

Comment:
```suggestion
```
Consider removing this for minimalist content. It's already said in the example heading, and the user doesn't need it to be repeated, imo.